indexer/validate_and_format_contacts.cpp

```diff
@@ -406,4 +416,97 @@ bool ValidateLinePage(string const & page)
   return (domain == "line.me" || strings::EndsWith(domain, ".line.me"));
 }
 
+bool isSocialContactTag(string_view tag)
+{
+  return tag == "contact:instagram" || tag == "contact:facebook" || tag == "contact:twitter"
+         || tag == "contact:vk" || tag == "contact:line";
+}
+
+bool isSocialContactTag(MapObject::MetadataID const metaID)
+{
+  return metaID == MapObject::MetadataID::FMD_CONTACT_INSTAGRAM ||
```

Comment:
nit: would the `switch` be better/faster?

Reply:
You mean replace:

```
  return metaID == MapObject::MetadataID::FMD_CONTACT_INSTAGRAM ||
         metaID == MapObject::MetadataID::FMD_CONTACT_FACEBOOK ||
         metaID == MapObject::MetadataID::FMD_CONTACT_TWITTER ||
         metaID == MapObject::MetadataID::FMD_CONTACT_VK ||
         metaID == MapObject::MetadataID::FMD_CONTACT_LINE;
```

with 

```
switch (metaID)
{
  case MapObject::MetadataID::FMD_CONTACT_INSTAGRAM:
  case MapObject::MetadataID::FMD_CONTACT_FACEBOOK:
  case MapObject::MetadataID::FMD_CONTACT_TWITTER:
  case MapObject::MetadataID::FMD_CONTACT_VK:
  case MapObject::MetadataID::FMD_CONTACT_LINE:
   return true;
  default:
    return false;
}
```
? I don't know how it would affect performance. We can try and use `switch`.

Reply:
That's not important. It's nit )